src/reuse/modules/service/odata.ts

```diff
@@ -197,14 +203,17 @@ export class OData {
    * };
    * let res = await service.odata.post(srv, "A_PurchaseOrder", payload);
    */
-  async post (srv: any, entitySet: string, payload: any) {
+  async post(srv: any, entitySet: string, payload: any, raw: boolean = false) {
     const entity = srv[entitySet];
     if (!entity) {
       throw new Error(`No entity set '${entitySet}' available in service`);
     }
-    return entity.post(payload);
+    if (raw) {
```

Comment:
better check for raw === true, otherwise it could be anything except for null, undefined and false
```suggestion
    if (raw === true) {
```